addon/src/lib/reducers/sideEffects.js

```diff
@@ -181,6 +183,20 @@ export function reducer(
         installManager.syncInstalled();
       };
 
+    case actions.BROWSER_STARTUP.type:
+      return ({ self, telemetry, getState }) => {
```

Comment:
@pdehaan thank you! not sure how that wasn't thrown when I ran the linter locally...